app/controllers/staffcos_controller.rb

```diff
@@ -1,5 +1,34 @@
+# frozen_string_literal: true
+
 class StaffcosController < ApplicationController
   def index
     @staffcos = Staffco.all
   end
+
+  def new
+    @staffcos = Staffco.new
+  end
+
+  def create
+    unless Staffco.exists?(staffco_params)
+
+    staffco = Staffco.create(staffco_params)
+    if staffco.persisted?
+      redirect_to staffcos_path, notice: 'The staffing company was successfully created'
+      #waiting for rsc can create jobs pr to include accessibility errors
+      # errors = user.errors.full_messages
+      # render json: { message: errors }, status: 422
+    end
```

Comment:
What if the object is NOT persisted? You need a `else` block to handle the exeption